runtime_scan/pkg/orchestrator/scanwatcher/watcher.go

```diff
@@ -538,9 +538,14 @@ func (w *Watcher) reconcileAborted(ctx context.Context, scan *models.Scan) error
 	scan.EndTime = utils.PointerTo(time.Now())
 	scan.State = utils.PointerTo(models.ScanStateFailed)
```

Comment:
Is the update on the `scan` object is needed?
Why not populating the `scanPatch` directly?
```
scanPatch := &models.Scan{
		State:     utils.PointerTo(models.ScanStateFailed),
		EndTime:   utils.PointerTo(time.Now()),
}
```

Reply:
> Is the update on the scan object is needed?